src/index.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly don't disable linters in your code. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Check this link for more information [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software)).